java/Oleksii-Riabov/.gitignore

```diff
@@ -0,0 +1,141 @@
+######################
+# Project Specific
+######################
+/build/www/**
+/src/test/javascript/coverage/
+/src/test/javascript/PhantomJS*/
+
+######################
```

Comment:
just wondering why do you need everything of this for the java project

Reply:
I take this .gitignore from John Thompson's course "Spring Framework5". I thought it will be good .gitignore file for most cases. What would you advise to add to .gitignore for current course?

Reply:
just remove everything that you don't use:
- project specific contains folders that are common for javascript projects
- node, SASS, ESLint sections are used for javascript projects as well - you don't need them
- Eclipse, Intellij, Visual Studio - you don't use all of them, so you can keep only one; same for operating systems sections

Usually, paths are added to gitignore when they are really needed; otherwise, it can become a mess

Reply:
And what about java projects, because I am on java course, not avascript)

Reply:
.gitignore has been updated. Thanks for advise!